ordersCostumes.js

```diff
@@ -0,0 +1,117 @@
+const addCostumeToOrder = async (pool, costumeId, orderId) => {
+    // throw error if costume id does not exist
+    const {rows:[costume]} = await pool.query(`
+        SELECT * FROM costumes
+        WHERE id = $1;
+    `, [costumeId])
+    if (costume === undefined) {
+        throw new Error(`Could not retrieve data because id provided for costume (${id}) does not exist in table.`)
+    } 
+
+    // throw error if order id does not exist
+    const {rows:[order]} = await pool.query(`
+        SELECT * FROM orders
+        WHERE id = $1;
+    `, [orderId])
+    if (order === undefined) {
+        throw new Error(`Could not retrieve data because id provided for order (${id}) does not exist in table.`)
+    } 
```

Comment:
Don't think this is necessary? I'm pretty sure it will throw an error if you call INSERT and don't pass both IDs. If it doesn't throw an error - make sure these are not allowed to be null in table definition

Reply:
It throws an error if the wrong ID is passed because of the foreign key constraint. It does not throw an error if no IDs are provided so I added the `NOT NULL` constraint as you suggested.